app/shared/directives/ipaTermSelectorDropdown/ipaTermSelectorDropdown.css

```diff
@@ -20,4 +20,16 @@
 /* Violates BEM styles to override bootstrap and neon */
 .btn.btn-default.ipa-term-selector-dropdown__button:hover {
 	border: 1px solid #777;
+}
+
+.btn-term-selector-inactive{
+	border: 1px solid #333;
```

Comment:
Fix indentation on line 26 to match the other lines (looks like a spaces vs tabs issue)

Reply:
fixed